src/migrations/201602231400_initial.js

```diff
@@ -53,6 +53,7 @@ function createSubscriptionsTable (knex) {
     table.string('event')
     table.string('subject', 1024)
     table.string('target', 1024)
+    table.boolean('is_deleted').index().defaultTo(false)
```

Comment:
Yeah, if it's going to be indexed at all, it should probably be a multi-column index. An index on a boolean field doesn't really make sense, since so many values repeat. I believe Oracle offers bitmapped indices for such cases, but knex doesn't support adding index type for anything besides postgres. So, presumably, this defaults to some kind of b-tree in oracle and sql server. (I tried to capture some of this info in the JIRA ticket.)